### ISSUE TITLE
[UI] Suggestion / Discussion

### ISSUE THREAD

**Issue by ghost** [loqui/im#406]
Hello all,

if you scroll a chat history with a person with a profil pic, you can see it in the small circle next to each msg received, and the pic of the sender too. 

Isent the big circle with the profil pic in the upper right enough?

**Comment by aesedepece** [comment]
Those small circles are put there just for consistency with group chats, in which those circles are quite useful for subconscious identification of each message's sender.

Do you think they are somewhat redundant in regular (one-to-one) chats?

**Comment by ghost** [comment]
yes, i think that it is somehow redundant in one-to-one chat. what is your opinion? and the opinion of other users?

off-topic: will you push an update with the xmpp corrections soon? we would like to re-start using xmpp on loqui!

**Comment by aesedepece** [comment]
Sure, as soon as XMPP MUC (chat rooms) are ready. Give us a week or so ;)

**Comment by ghost** [comment]
Thanks!

well my suggestion is to use the small circle only in groups chats. what do you think about that?

**Comment by aesedepece** [comment]
I'd like to hear other users' opinion before making any decision. Let's discuss it.
@Gioyik please have your say!

P.S.: Feel free to post any other usability issue or suggestion that you have. Sometimes we think something to be obvious but then users have the last word :wink:

**Comment by Gioyik** [comment]
Mmmm I am not sure about this. I think is so early to take a choice, and I see those small circles in the right way. Maybe we need more feedback related to this and see if it's redundant. The v0.3.0 is so new. We could take a time to discuss and see the comments of more people.
